ng2-components/ng2-activiti-tasklist/src/components/activiti-people-search.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="mdl-textfield mdl-js-textfield mdl-textfield--floating-label">
+    <input class="mdl-textfield__input" type="text" id="userSearchText" [value]=""
+           [formControl]="searchUser"/>
+    <label class="mdl-textfield__label" for="userSearchText">Search user</label>
+</div>
+<ul class='mdl-list'>
+    <li class="mdl-list__item fix-element-user-list" *ngFor="let user of userList">
+        <span class="mdl-list__item-primary-content mdl-button mdl-js-button mdl-js-ripple-effect"
+              (click)="onRowClick(user)" id="user-{{user.id}}">
+            <i class="material-icons md-light">face</i>
+            {{ user.firstName }} - {{ user.lastName }}
```

Comment:
Activiti sometimes returns 'null' (as string) value for first/last name. It is better creating a method `getDisplayName(user: <yourType>): string` and evaluating all cases there, see Activiti Form component implementation.